lib/membrane_mp4/movie_fragment_box.ex

```diff
@@ -24,15 +24,21 @@ defmodule Membrane.MP4.MovieFragmentBox do
   """
   alias Membrane.MP4.Container
 
-  @trun_flags %{data_offset: 1, sample_duration: 0x100, sample_size: 0x200, sample_flags: 0x400}
+  @trun_flags %{
+    data_offset: 1,
+    sample_duration: 0x100,
+    sample_size: 0x200,
+    sample_flags: 0x400
+  }
   @mdat_data_offset 8
 
   @spec assemble(%{
           sequence_number: integer,
           elapsed_time: integer,
           timescale: integer,
           duration: integer,
-          samples_table: [%{sample_size: integer, sample_flags: integer}]
+          samples_table: [%{sample_size: integer, sample_flags: integer}],
+          sample_composition_time_offsets_present: true | false
```

Comment:
1. The config accepted as parameter does not have this flag, it is added inside the function
2. What is there, is `support_b_frames` flag
3. `true | false` is the definition of `boolean`

```suggestion
          support_b_frames: boolean
```

Reply:
done